index.js

```diff
@@ -52,4 +53,135 @@ const state = {
     }
   ],
   cart: []
-};
\ No newline at end of file
+};
+
+// SELECTORS
+const storeItemList = document.querySelector('.store--item-list')
+const cartItemList = document.querySelector('.cart--item-list')
+const totalNumber = document.querySelector('.total-number')
+
+
+const clearStorePage = () => {
```

Comment:
Good idea using functions to clear the store/cart, definitely a more elegant solution